lib/kinde_sdk.rb

```
@@ -70,35 +72,56 @@
+    def refresh_token(hash,
+      client_id: @config.client_id,
+      client_secret: @config.client_secret,
+      audience: "#{@config.domain}/api",
+      domain: @config.domain
+    )
+      OAuth2::AccessToken.from_hash(@config.oauth_client(
+        client_id: client_id, 
+        client_secret: client_secret,
+        domain: domain,
+        authorize_url: "#{domain}/oauth2/auth",
+        token_url: "#{domain}/oauth2/token"), hash).refresh.to_hash
```

RuboCop (1.69.1)

[convention] Indent the first argument one step more than `@config.oauth_client(`.

(Layout/FirstArgumentIndentation)